domain_coordinator/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>domain_coordinator</name>
+  <version>0.1.0</version>
+  <description>A tool to coordinate unique ROS_DOMAIN_IDs across multiple processes</description>
+  <maintainer email="pete.baughman@apex.ai">Pete Baughman</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+
+  <test_depend>ament_cmake_pytest</test_depend>
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+
+  <export>
+    <build_type>ament_cmake</build_type>
```

Comment:
Could be move this to be an `ament_python` package?
For installing `domain_coordinator/cmake/run_test_isolated.py` you could maybe use `data_files` argument in `setup.py` `setup` call, or add a console_script entry_point.

Reply:
Maybe, but then I'm not sure how I would look up the path [here](https://github.com/ros2/ament_cmake_ros/pull/3/files#diff-7f2b107d52b58d499258c6389dd603b5R22) and [here](https://github.com/ros2/ament_cmake_ros/pull/3/files#diff-c144bdd25aa07f188f93aa3dfc17b8a5R22)